internal/webhookhandler/webhookhandler_test.go

```diff
@@ -403,15 +403,17 @@ func TestFailOnInvalidRequest(t *testing.T) {
 		name     string
 		req      admission.Request
 		expected int32
+		allowed  bool
 	}{
 		{
-			"empty payload",
-			admission.Request{},
-			http.StatusBadRequest,
+			name:     "empty payload",
+			req:      admission.Request{},
+			expected: http.StatusBadRequest,
+			allowed:  false,
```

Comment:
Now the webhook fails the pod creation only if it cannot decode the request, which should not happen in practice.